grassroots-backend/src/phone-canvass/Scheduler/PhoneCanvassScheduler.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable grassroots/entity-use */
```

Comment:
I see this turned off in a number of files. Is there something wrong with the rule? Should we disable it globally?
Usually I prefer having linter exceptions at the line level so it's easier to see and remove them in the future.